flowzone.yml

```diff
@@ -1137,6 +1189,13 @@ jobs:
       <<: *gitHubCliEnvironment
 
     steps:
+
+      - *getHeadBranchMergeCommits
+      - *rejectNonLinearHead
```

Comment:
These seem like they must be run together and in order so I'm wondering why they aren't a single anchor since they are so 1-1 dependent?